src/library/parserm3u.cpp

```diff
@@ -107,10 +107,24 @@ bool ParserM3u::writeM3UFile(const QString &file_str, const QList<QString> &item
     QString fileContents(QStringLiteral("#EXTM3U\n"));
     for (const QString& item : items) {
         fileContents += QStringLiteral("#EXTINF\n");
-        if (useRelativePath) {
-            fileContents += baseDirectory.relativeFilePath(item) + QStringLiteral("\n");
+        QByteArray trackByteArray = QTextCodec::codecForName(kStandardM3uTextEncoding)
+                                            ->fromUnicode(item);
+        if ((trackByteArray == item) || useUtf8) {
```

Comment:
This line requires a source code comment comment. It uses QString::toUtf8() see: https://doc.qt.io/qt-5/qbytearray.html#operator-eq-eq

So you compare the "Windows-1250" encoding with the Utf8 encoding. I think they match for all ASCII characters, but not for Ä and friends. 
You approach should work when using QTextCodec::toUnicode(trackByteArray); 

It would be also nice if the check is bypassed when useUtf8 is set.

Reply:
Thanks for your review. 

I will work on it :+1: